app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala

```diff
@@ -126,17 +126,16 @@ class EditSetRecordController @Inject() (
         for {
           placesOfDeposit <- referenceDataService.getPlacesOfDeposit()
           legalStatuses   <- referenceDataService.getLegalStatuses
-        } yield (placesOfDeposit, legalStatuses) match {
-          case (placesOfDeposit, legalStatuses) =>
-            BadRequest(
-              generateEditSetRecordEditView(user, editSet, record, placesOfDeposit, legalStatuses, formWithErrors)
-            )
-        }
+          creators        <- referenceDataService.getCreators()
+        } yield BadRequest(
+          generateEditSetRecordEditView(user, editSet, record, placesOfDeposit, creators, legalStatuses, formWithErrors)
+        )
       case Left(EditSetNotFound(missingEditSetId)) =>
         BadRequest(s"Edit Set with ID [$missingEditSetId] not found")
       case Left(EditSetRecordNotFound(missingOci)) => BadRequest(s"Record with $missingOci not found")
       case Left(InvalidAction(badAction))          => BadRequest(s"$badAction is not allowed action")
       case Left(MissingAction)                     => BadRequest("This action is not allowed")
+      case Left(_)                                 => BadRequest("Unknown action")
```

Comment:
A good addition, but perhaps we should have a scenario that triggers it?

Reply:
We spoke about this, and it would be more trouble than it's worth - at least how it's currently implemented.

I'm OK with postponing this.